setup.py

```diff
@@ -1,9 +1,28 @@
 from setuptools import setup, find_packages
 
+
+def read_version(version_file):
+    """
+    Given the input version_file, this function extracts the
+    version info from the __version__ attribute.
+    """
+    version_str = None
+    import re
+    verstrline = open(version_file, "rt").read()
+    VSRE = r"^__version__ = ['\"]([^'\"]*)['\"]"
+    mo = re.search(VSRE, verstrline, re.M)
+    if mo:
+        version_str = mo.group(1)
+    else:
+        raise RuntimeError("Unable to find version string in %s." % (version_file,))
+    return version_str
+
 setup(
     name='RLTest',
-    version='0.2.1',
+    version=read_version("RLTest/_version.py"),
```

Comment:
Can we import `RLTest/_version.py` and use `__version__` like we do in other Python modules?

Reply:
you're totally right Rafi. Addressed the change in latest commit.